terraform-apply.sh

```diff
@@ -1,38 +1,23 @@
 #!/bin/bash
 # vim: set ft=sh
 
-set -ex
-
-if [ -z "$STACK_NAME" ]; then
```

Comment:
Because I added `set -u` above. I'll switch back if you like the current checks better.